mscore/plugin/mscorePlugins.cpp

```diff
@@ -438,7 +438,7 @@ void MuseScore::pluginTriggered(QString pp)
                   }
             }
 
-      connect(engine, &QmlPluginEngine::endCmd, p, &QmlPlugin::endCmd);
+      engine->mapPluginSignals(p);
```

Comment:
For consistency and future development, this connect and [this one](https://github.com/musescore/MuseScore/pull/5675/files#diff-46d4763e789ee0d1bb3401cbebf030a3L375) have been refactored into a single function.